drivers/flash/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ if(CONFIG_CLOCK_CONTROL_STM32_CUBE)
   zephyr_sources_ifdef(CONFIG_SOC_SERIES_STM32F7X flash_stm32f7x.c)
```

Comment:
Have you been able to test flash driver?
(using sample/drivers/flash_shell)

Reply:
Testing samples/drivers/flash_shell  --> PASSED

uart:~$ flash erase 0x1E000 0x1000
uart:~$ flash write 0x1E000 00 01 02 03 04 05 06 07
Reading back written bytes:
00 01 02 03 | 04 05 06 07
uart:~$ flash read 0x1E000 10
00 01 02 03 | 04 05 06 07
ff ff